### ISSUE TITLE
Need to manually add external package folder to PsychoPy directory, add paths to preferences doesn't work

### ISSUE THREAD

**Issue by tian-lan** [psychopy/psychopy#3162]
When adding external Python packages, such as scikit-learn and MNE, the entire package folder needs to be downloaded and placed under ‘ **…/PsychoPy3/Lib/site-packages/** ’ in order to work. Previous instructions on adding packages via [adding paths to preferences](https://www.psychopy.org/recipes/addCustomModules.html) won't work any more. 

This might be an existing issue since Python 2.7. 

Thanks.

**Comment by peircej** [comment]
You mention Py 2.7. Is the package you're trying to install from 2.7?

**Comment by tian-lan** [comment]
Sorry for the confusion, let me rephrase...
I mentioned Python 2.7 since [Todd mentioned that this might be an issue since Py 2.7](https://discourse.psychopy.org/t/psychopy-does-not-recognize-scikit-learn/16450/4).
I'm using installing packages from 3.6.

Thanks.

**Comment by EitanHemed** [comment]
Is this solved by the package installation feature introduced in 2023.1.0? (**[Plugins and Packages](https://github.com/psychopy/psychopy/releases/tag/2023.1.0)**)

**Comment by TEParsons** [comment]
Sort of, yes! If the necessary packages are part of a plugin, then they'll be installed along with the plugin. Otherwise, the Packages tab allows for pip install (though, you may need to open the app as admin in order to install to protected folders)
